jablib/src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -1456,7 +1457,7 @@ public InternalPreferences getInternalPreferences() {
                 Version.parse(get(VERSION_IGNORED_UPDATE)),
                 getBoolean(VERSION_CHECK_ENABLED),
                 getPath(PREFS_EXPORT_PATH, getDefaultPath()),
-                getUserAndHost(),
+                getUserAndHost().toString(),
```

Comment:
toString returns internal jdk string representation. this is outside of your control, use custom getCanonnicalForm() method instead.